samtranslator/model/api/api_generator.py

```diff
@@ -513,16 +513,16 @@ def _construct_api_domain(self, rest_api, route53_record_set_groups):  # type: i
             basepath_mapping.Stage = ref(rest_api.logical_id + ".Stage")
             basepath_resource_list.extend([basepath_mapping])
         else:
-            for path in basepaths:
-                path = "".join(e for e in path if e.isalnum())
+            for basepath in basepaths:
+                path = "".join(e for e in basepath if e.isalnum())
```

Comment:
What changed? Any tests?

Reply:
oh wait, the test file changes didn't got committed. Let me add that